lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_lint_localizations.rb

```diff
@@ -2,10 +2,20 @@ module Fastlane
   module Actions
     class IosLintLocalizationsAction < Action
       def self.run(params)
-        violations = {}
+        violations = Hash.new([])
 
         loop do
-          violations = self.run_linter(params)
+          # If we did `violations = self.run...` we'd lose the default value for missing key being `[]` that we set above with `Hash.new`.
+          # We want that default value so that we can use `+=` when adding the duplicate keys violations below.
+          violations = violations.merge(self.run_linter(params))
```

Comment:
Alternate syntax (just for style nitpicking):

```suggestion
          violations.merge!(self.run_linter(params))
```